target/linux/rockchip/image/armv8.mk

```diff
@@ -2,6 +2,34 @@
 #
 # Copyright (C) 2020 Tobias Maedel
 
+define Device/friendlyarm_nanopi-m4-2gb
+  DEVICE_VENDOR := FriendlyARM
+  DEVICE_MODEL := NanoPi M4 2GB
+  DEVICE_DTS := rockchip/rk3399-nanopi-m4
+  SOC := rk3399
+  SUPPORTED_DEVICES :=friendlyarm,nanopi-m4
+  UBOOT_DEVICE_NAME := nanopi-m4-2gb-rk3399
+  IMAGE/sysupgrade.img.gz := boot-common | boot-script | pine64-img | gzip | append-metadata
+  DEVICE_PACKAGES := \
+	brcmfmac-firmware-4356-sdio \
+	brcmfmac-firmware-4356-sdio-nanopi-m4 \
+	kmod-bluetooth kmod-brcmfmac wpad-basic-wolfssl
+endef
+TARGET_DEVICES += friendlyarm_nanopi-m4-2gb
+
+define Device/friendlyarm_nanopi-m4
+  DEVICE_VENDOR := FriendlyARM
+  DEVICE_MODEL := NanoPi M4
```

Comment:
If the other one is "2GB", this one should have an identifier, too.
Apart from that, consider using DEVICE_VARIANT variable for that.